componets/snackBar/snackBar.css

```diff
@@ -0,0 +1,41 @@
+.snackbar {
+	display: flex;
+	align-items: center;
+	border-radius: 5px;
+	padding: 8px;
+	justify-content: space-around;
+	position: fixed;
+	bottom: 5rem;
+	background-color: var(--text-color);
+	color: var(--background-color);
+	font-size: 1rem;
+	display: none;
+	z-index: 10;
+}
+.snackbar p {
+	display: inline-flex;
+}
+.btn__snackBar {
+	border: none;
+	margin: 0 1rem;
+	background: none;
+	color: var(--secondary-color);
+	font-size: 1rem;
+	cursor: pointer;
+}
+.btn__snackBar:hover {
+	color: var(--primary-color);
+}
+i {
+	padding: 4px;
+	cursor: pointer;
+}
+
+i:hover {
+	background-color: var(--box--shadow-color);
+	border-radius: 50%;
+}
```

Comment:
Directly referencing `HTML Tag` may cause problem while using in other projects. If it is common for `i` tag element , then you can move to **main css file**

Reply:
Thanks for pointing this out, I have made the changes added a class for it.